src/routes/index.ts

```diff
@@ -349,7 +349,7 @@ router.route(config.UPDATE_BENEFICIAL_OWNER_TYPE_URL)
     navigation.hasEntityUpdateDetails
   )
   .get(updateBeneficialOwnerType.get)
-  .post(...validator.beneficialOwnersType, checkValidations, updateBeneficialOwnerType.post);
+  .post(...validator.updateBeneficialOwnerAndManagingOfficerType, checkValidations, updateBeneficialOwnerType.post);
```

Comment:
Did you run this past Matthew at all? Temporary navigation for the BO/MO has been added as hasUpdatePresenter - with the view that it will change once we introduce the initial page for this section of the journey.